modules/installation-mirror-repository.adoc

```diff
@@ -22,38 +22,41 @@ Complete the following steps on the bastion host:
 . Review the
 link:https://access.redhat.com/downloads/content/290/[{product-title} downloads page]
 to determine the version of {product-title} that you want to install.
+The image release advisory includes the pullspec repository and digest, for example link:https://access.redhat.com/errata/product/290/ver=4.3/rhel---7/x86_64/RHBA-2020:0062[`quay.io/openshift-release-dev/ocp-release` and `sha256:3a516480dfd68e0f87f702b4d7bdd6f6a0acfdac5cd2e9767b838ceede34d70d` for the x86-64 4.3.0 release image].
 
 . Set the required environment variables:
 +
 ----
-$ export OCP_RELEASE=<release_version> <1>
-$ export LOCAL_REGISTRY='<local_registry_host_name>:<local_registry_host_port>' <2>
-$ export LOCAL_REPOSITORY='<repository_name>' <3>
-$ export PRODUCT_REPO='openshift-release-dev' <4>
+$ export RELEASE_REPOSITORY='<release_repository>' <1>
+$ export RELEASE_DIGEST='<release_digest>' <2>
+$ export LOCAL_REGISTRY='<local_registry_host_name>:<local_registry_host_port>' <3>
+$ export LOCAL_REPOSITORY='<repository_name>' <4>
 $ export LOCAL_SECRET_JSON='<path_to_pull_secret>' <5>
-$ export RELEASE_NAME="ocp-release" <6>
+$ export LOCAL_TAG='<local_tag_name>' <6>
 ----
-<1> For `<release_version>`, specify the version number of {product-title} to
-install, such as `4.3.0`.
-<2> For `<local_registry_host_name>`, specify the registry domain name for your mirror
+<1> For `<release_repository>`, specify the repository pullspec for the source
+release image.  For example `quay.io/openshift-release-dev/ocp-release`.
+<2> For `<release_digest>`, specify the digest from the release image advisory,
+such as `sha256:3a516480dfd68e0f87f702b4d7bdd6f6a0acfdac5cd2e9767b838ceede34d70d`.
+<3> For `<local_registry_host_name>`, specify the registry domain name for your mirror
 repository, and for `<local_registry_host_port>`, specify the port that it
 serves content on.
-<3> For `<repository_name>`, specify the name of the repository to create in your
+<4> For `<repository_name>`, specify the name of the repository to create in your
 registry, such as `ocp4/openshift4`.
-<4> The repository to mirror. For a production release, you must specify
-`openshift-release-dev`.
 <5> For `<path_to_pull_secret>`, specify the absolute path to and file name of
 the pull secret for your mirror registry that you created.
-<6> The release mirror. For a production release, you must specify
-`ocp-release`.
+<6> For `<local_tag_name>`, specify an image tag compatible with your
+local registry.  The tag value has no impact on OpenShift; its purpose is to
+keep your local registry from garbage-collecting the mirrored image.
```

Comment:
```suggestion
keep your local registry from deleting the mirrored image.
```

I'm not attached to this suggestion, but "garbage-collecting" won't translate well.